packages/globals/lib/index.d.ts

```diff
@@ -49,6 +49,7 @@ interface PlatformaticGlobalInterface {
   setOpenapiSchema(schema: object): void
   setGraphqlSchema(schema: object): void
   setConnectionString(connection: string): void
+  setCustomHealthCheck(healthCheck: Function): void
```

Comment:
Here and in the other file I would use a more accurate signature for the function, something like:

```
setCustomHealthCheck(healthCheck: () => boolean | Promise<boolean>): void
```